### ISSUE TITLE
Modify the learning rate according to known relationships

### ISSUE THREAD

**Issue by cosmaadrian** [cosmaadrian/acumen-template#22]
What is the best way to change the LR for

- Scaling batch size [https://openreview.net/pdf?id=F2mhzjHkQP](https://www.cs.princeton.edu/~smalladi/blog/2024/01/22/SDEs-ScalingRules/)
- Scaling model width https://arxiv.org/pdf/2203.03466 
- Scaling model depth https://openreview.net/pdf?id=17pVDnpwwl 
- Scaling token horizon https://arxiv.org/pdf/2409.19913 

What is the best way to change the weight decay for

- scaling batch size
- Scaling model width
- Scaling model depth
- Scaling sequence length
- Scaling token horizon

**Comment by cosmaadrian** [comment]
Check previous projects
